web/src/popover_menus.js

```diff
@@ -1023,7 +1023,7 @@ export function initialize() {
         },
         onHidden(instance) {
             instance.destroy();
-            popover_instances.all_messages = undefined;
+            popover_instances.mark_all_messages_as_read = undefined;
```

Comment:
The template and `render_all_messages_sidebar_actions` probably also want to be renamed.

I think rather than `mark_all_messages_as_read` as a name, which makes it hard to add other things in the future, maybe `left_sidebar_inbox_popover` would be a good long-term name for this?

Reply:
Changed and pushed.